app/features/dynamo/tools.py

```diff
@@ -1,20 +1,20 @@
 from langchain_community.document_loaders import YoutubeLoader
 from langchain_text_splitters import RecursiveCharacterTextSplitter
 from langchain.prompts import PromptTemplate
-from langchain_google_genai import GoogleGenerativeAI
+from langchain_google_vertexai import VertexAI
```

Comment:
Why is `langchain_google_genai.GoogleGenerativeAI` being changed to `langchain_google_vertexai.VertexAI` in another feature, while you're still using `langchain_google_genai.GoogleGenerativeAI` in your syllabus generator?